Sources/SourceKitLSP/Swift/Diagnostic.swift

```diff
@@ -19,8 +19,8 @@ extension CodeAction {
 
   /// Creates a CodeAction from a list for sourcekit fixits.
   ///
-  /// If this is from a note, the note's description should be passed as `fromNote`.
-  init?(fixits: SKDResponseArray, in snapshot: DocumentSnapshot, fromNote: String?) {
+  /// If this is from a notification, the notification's description should be passed as `fromNotification`.
+  init?(fixits: SKDResponseArray, in snapshot: DocumentSnapshot, fromNotification: String?) {
```

Comment:
This file is about diagnostics, which do indeed have notes, not notifications. Could you change it back?

Reply:
I changed these back. Let me know if this works!